about-us.html

```diff
@@ -171,24 +171,51 @@ <h2 >Community Goals</h2>
+                <div class="section-title text-center">
+                    <h2>Projects</h2>
+                    <p class="customParagraphStyle">Sugar Labs is working on a few <a href="https://wiki.sugarlabs.org/go/Category:Project" class="hrefCustomColor">projects</a> focused on delivering specific goals in a defined time period. Below are some of our premier projects:</p>
+                </div>
+            </div>
+        </div>
         <div class="row">
-            <div class="col-md-12" >
-                <div class="row justify-content-center">
-                    <div class="col-md-8 col-md-push-2" >
-                        <div class="section-title text-center">
-                            <h2>Projects</h2>
-                            <p class="customParagraphStyle">Sugar Labs is working on a few <a href="https://wiki.sugarlabs.org/go/Category:Project" class="hrefCustomColor">projects</a> focused on delivering specific goals in a defined time period. The "Projects" panel in the Sugar Labs wiki sidebar holds our premier projects. Please also see these project home pages:
-                            <ul>
-                                <li><a href="https://wiki.sugarlabs.org/go/Sugar_Creation_Kit" class="hrefCustomColor">Sugar Creation Kit</a></li>
-                                <li><a href="https://wiki.sugarlabs.org/go/Sugar_on_a_Stick" class="hrefCustomColor">Sugar on a Stick</a></li>
-                                <li><a href="https://wiki.sugarlabs.org/go/Math4Team" class="hrefCustomColor">Math4 Project</a></li>
-                                <li><a href="https://wiki.sugarlabs.org/go/Summer_of_Code" class="hrefCustomColor">Summer of Code</a></li>
-                            </ul>
-                            </p>  
-                        </div>
-                    </div>
+            <div class="col-sm-6 col-md-6 mb-4">
+                <div class="project-card">
+                    <h4>Sugar Creation Kit</h4>
+                    <p>A toolkit to help users create engaging and interactive content for the Sugar environment.</p>
+                    <a href="https://wiki.sugarlabs.org/go/Sugar_Creation_Kit">
+                        <button type="button" class="btn btn-raised btn-custom">Learn More</button>
+                    </a>
+                </div>
+            </div>
+            <div class="col-sm-6 col-md-6 mb-4">
+                <div class="project-card">
+                    <h4>Sugar on a Stick</h4>
+                    <p>A portable version of Sugar that can run from a USB stick without installation.</p>
+                    <a href="https://wiki.sugarlabs.org/go/Sugar_on_a_Stick">
+                        <button type="button" class="btn btn-raised btn-custom">Learn More</button>
+                    </a>
+                </div>
+            </div>
+            <div class="col-sm-6 col-md-6 mb-4">
+                <div class="project-card">
+                    <h4>Math4 Project</h4>
```

Comment:
I have no sense of whether or not this project is still alive.

It may be good to comment this part out for now...